### PR TITLE
Evolved App UI and Data Output

### PR DESCRIPTION
**Pull request by DanielNMartin**
@aaron-goshine  
@stevestotter       Please review latest changes. :)

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -3,28 +3,51 @@
...
+        text-decoration: none;
+        display: inline-block;
+        font-size: 16px;
+        margin: 4px 2px;
+        cursor: pointer;
+        border-top: 10px;
 
       }
 
       .Start_Date {
```
Comment (stevestotter) [inline index.html]: I would stay consistent in your naming of variables - i.e. using camel case everywhere or underscores everywhere. You already have "bodySection" as a class name elsewhere - I would stick to camelCase.

File: index.html [hunk 2]
```diff
@@ -3,28 +3,51 @@
 
 <head>
  <Title>Date Range Analyser</Title>
- <script src="./index.js">
- </script>
+
 
  <style>
```
Comment (stevestotter) [inline index.html]: Styles in the same file? Why don't you have another file called styles.css and include it on this page instead as a resource?

File: index.js [hunk 3]
```diff
@@ -1,22 +1,32 @@
 
-function dateRangeAnaliser(startDate, endDate, step ) {
-var listDate = [];
-while (startDate.getTime() < endDate.getTime()) {
-  startDate.setDate(startDate.getDate() + step);
-  var tempDate = new Date(startDate.getTime());
-  var stringDate = tempDate.getDate() + ", " + (tempDate.getMonth() + 1) + ", " + tempDate.getFullYear() + " "
-  listDate.push(stringDate);
-};
+function dateRangeAnaliser (startDate, endDate, step) {
```
Comment (stevestotter) [inline index.js]: Analyser*

**Review by stevestotter — CHANGES_REQUESTED** [review]
Needs a couple of changes, mainly semantic. Then add some tests in the next PR?

File: index.html (hunk 2 above)
Comment (DanielNMartin) [reply index.html]: Styles.css file added to project.

File: index.html (hunk 1 above)
Comment (DanielNMartin) [reply index.html]: Edited code for variable naming consistency.

File: index.js (hunk 3 above)
Comment (DanielNMartin) [reply index.js]: Amended this typo across my code.

**Comment by DanielNMartin** [comment]
@stevestotter  
Styles file added as requested and all semantical errors have been amended. 
Please re-review.

**Comment by stevestotter** [comment]
LGTM 👍

**Pull request merged**